README.md

```diff
@@ -33,6 +33,7 @@ jobs:
       env:
         GH_BOT_TOKEN: ${{ secrets.GH_BOT_TOKEN }}
         SKIP_FOLDERS: "tests,.github"
+        SKIP_LINT: "no"
```

Comment:
It would be better if you change the env. variable to `PHP_LINT`, to make it more simple and in line with the `vip-go-ci` flag `--lint`.
Setting it to `true` by default in the code when no value is found. As it will have a default value, let us not add it in the main example of the readme. Information about it in the `#environment-variables` section is good enough.